frontend/talentsearch/src/js/components/browse/BrowsePoolsPage.test.tsx

```diff
@@ -0,0 +1,172 @@
+/**
+ * @jest-environment jsdom
+ */
+import "@testing-library/jest-dom";
+import { render, screen, act } from "@testing-library/react";
+import React from "react";
+import { IntlProvider, MessageFormatElement } from "react-intl";
+import { axeTest } from "@common/helpers/testUtils";
+import { BrowsePools, BrowsePoolsProps } from "./BrowsePoolsPage";
+import {
+  AdvertisementStatus,
+  PoolAdvertisement,
+  PublishingGroup,
+} from "../../api/generated";
+
+const publishedItJobsPool: PoolAdvertisement = {
+  id: "publishedItJobsPool",
+  publishingGroup: PublishingGroup.ItJobs,
+  advertisementStatus: AdvertisementStatus.Published,
+};
+
+const expiredItJobsPool: PoolAdvertisement = {
+  id: "expiredItJobsPool",
+  publishingGroup: PublishingGroup.ItJobs,
+  advertisementStatus: AdvertisementStatus.Expired,
+};
+
+const archivedItJobsPool: PoolAdvertisement = {
+  id: "archivedItJobsPool",
+  publishingGroup: PublishingGroup.ItJobs,
+  advertisementStatus: AdvertisementStatus.Archived,
+};
+
+const publishedExecJobsPool: PoolAdvertisement = {
+  id: "publishedExecJobsPool",
+  publishingGroup: PublishingGroup.ExecutiveJobs,
+  advertisementStatus: AdvertisementStatus.Published,
+};
+
+const renderWithReactIntl = (
+  component: React.ReactNode,
+  locale?: "en" | "fr",
+  messages?: Record<string, string> | Record<string, MessageFormatElement[]>,
+) => {
+  return render(
+    <IntlProvider locale={locale || "en"} messages={messages}>
+      {component}
+    </IntlProvider>,
+  );
+};
```

Comment:
If you import `render` from `testUtils` it does a similar thing, I believe.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/73d18d877abe9b2d1cfc4cfc52bc0234d7b52cc0/frontend/talentsearch/src/js/tests/testUtils.tsx#L6-L13

Reply:
Oh, I forgot about that, thanks!  Updated in [Remove custom render](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4268/commits/15d8fef88536855f1c998416500900674ce6c9e9).